lib/task-data/tasks/validate-ssh.js

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+
+module.exports = {
+    friendlyName: 'Validate Ssh',
+    injectableName: 'Task.Ssh.Validation',
+    implementsTask: 'Task.Base.Ssh.Validation',
+    options: {
+        timeout: null,
+        retries: null,
+        backoffDelay: null,
+        users: null
```

Comment:
Will you add the root variables here too, so a graph definition can fill them in with "defaults" ?